tabs/src/components/sample/style/TabConfig.css

```diff
@@ -0,0 +1,24 @@
+.caption {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  flex-grow: 1;
+  height: 40vh;
+  width: 100vw;
+  background-color: #fff;
+  position: relative;
+  font-size: 110%;
```

Comment:
Why is the font-size set to 110%? Feels like we're going directly against the standards that would be set in the original component if we override the font-size? 

Also let's not override the background color in case Northstar supports themes!

Reply:
I'm not sure why font-size is 110%, I think @gracet9182 added that. For background color I can remove that, unless there are any objections from @gracet9182 ?

Reply:
Oh I didn't think it mattered since I was seeing things like negative margins in other places before - feel free to remove the font size. The background color change was because it was inheriting a `*` background colour change from `index.css` body. I have no objections to any css changes but this was just fixed the problem

Reply:
https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/31/commits/dc48d79692adeccbbec48ff0d20792cb6b06b718

Reply:
Looking at the tab, I think I need to keep the background color because it's grey otherwise.
![image](https://user-images.githubusercontent.com/55033656/127515883-af207521-6f31-4b4f-8844-3fdcf4d2a89c.png)

Reply:
![Screenshot 2021-07-29 111830](https://user-images.githubusercontent.com/47228306/127545329-7f1109a5-e4ff-4618-819c-5c1e082efd55.png)